### ISSUE TITLE
Thanks for the widget

### ISSUE THREAD

**Issue by KOT040188** [tubbadu/Scriptinator#21]
The widget is good, but not quite suitable for my tasks. How about the idea of ​​making a widget with a pop-up menu, this menu will have predefined commands, they will be executed when clicked.

**Comment by Sadi58** [comment]
Indeed, you can configure Scriptinator exactly the way you want, i.e. click to open a popup menu including items to execute various commands (apps, scripts) when clicked.
All you need is a script powered by YAD (Yet Another Dialog), which is like an advanced Zenity, that will be set as both "Init script" and "OnClick script" (perhaps leaving "Periodic script" blank).
You can find a sample YAD script for this here: https://eirenicon.org/knowledge-base/yad-yet-another-dialog/
You only need to adapt the one for "Free Office Access" to your needs.

**Comment by tubbadu** [comment]
> The widget is good, but not quite suitable for my tasks. How about the idea of ​​making a widget with a pop-up menu, this menu will have predefined commands, they will be executed when clicked. There are many such widgets, but not one of them can be placed in the tray.

Thanks man! Just adding to @Sadi58's awesome reply that if there's another plasmoid that you like and want to place it in the tray you only have to modify the desktop file located at `~/.local/share/plasma/plasmoids/plasmoidnamehere/metadata.desktop` appending the line 
```desktop
X-Plasma-NotificationArea=true
```
